test/unit/test_deep_learning_container.py

```diff
@@ -14,11 +14,10 @@
 
 import unittest
 
+from docker.build_artifacts import deep_learning_container as deep_learning_container_to_test
```

Comment:
this import should go after pytest and request because it's our library

Reply:
Had to do this to comply with flake8 requirements.
I think module names need to be sorted in lexicographical order.

Reply:
I actually think the reason this is happening is b/c flake8 recognizes "docker" as the docker python sdk, so it thinks this is a 3rd party lib instead of a local one. One possible workaround is to noqa this line for that particular flake8 rule.